### ISSUE TITLE
Twg_WageSettings_Staff.LoadSettings

### ISSUE THREAD

**Issue by gsbelarus** [GoldenSoftwareLtd/gedemin#2079]
Originally reported on Google Code with ID 2079

```
там есть строчки
 DischargeDepKey = F.ReadInteger("DischargeDepKey", gdcBaseManager.GetIDByRUIDString("147080894_1224850260"))

DischargeDepKey - подразделение для уволенных сотрудников. Оно находится в организации
Зарплата - эталон. А если сотрудник принадлежит другой организации, он "исчезает"
```

Reported by `alexandra.gsoftware` on 2010-07-09 11:49:53

**Comment by gsbelarus** [comment]
```
Он принадлежит стартовой организации, а вообще там в настройках можно же указывать своё
подразделение.
```

Reported by `Alexander.GoldenSoft` on 2010-07-09 11:58:47
- **Status changed**: `Invalid`

**Comment by gsbelarus** [comment]
Reported by `gs1994` on 2010-07-12 10:35:39
- **Status changed**: `Duplicate`
- **Merged into**: #2080
